docroot/profiles/humsci/su_humsci_profile/su_humsci_profile.install

```diff
@@ -931,3 +932,62 @@ function su_humsci_profile_update_9709() {
   \Drupal::service('module_installer')->install(['paragraphs_features']);
   \Drupal::service('module_installer')->install(['paragraphs_ee']);
 }
+
+/**
+ * Uninstall and remove config for hs_row paragraph type and modules.
+ */
+function su_humsci_profile_update_9710() {
+  // Remove existing paragraph entities.
+  $paragraph_storage = \Drupal::entityTypeManager()->getStorage('paragraph');
+  $paragraph_machine_name = 'hs_row';
+
+  $query = $paragraph_storage->getQuery()
+    ->condition('type', $paragraph_machine_name)
+    ->accessCheck(FALSE);
+  $paragraph_ids = $query->execute();
+
+  if ($paragraph_ids) {
+    $paragraphs = Paragraph::loadMultiple($paragraph_ids);
+    foreach ($paragraphs as $paragraph) {
+      $paragraph->delete();
+    }
+
+    $message = t('Deleted all paragraphs of type @type.', [
+      '@type' => $paragraph_machine_name,
+    ]);
+    Drush::logger()->notice($message);
+  }
```

Comment:
I'm not exactly sure how many `hs_row` paragraphs there are out in the wild and this might be better in its own database update with a `$sandbox` batch to be on the safe side

Reply:
Done.